lib/fastlane/actions/hockey.rb

```diff
@@ -19,6 +19,10 @@ def self.run(options)
         if options[:dsym]
           dsym_filename = options[:dsym]
         else
+          if options[:ipa].to_s.length == 0
+            UI.user_error!("You have to provide an ipa file")
```

Comment:
in this case it must be an ipa file, only if the `upload_dsym_only` is set, a dSYM is enough. The most common use case is to upload a binary though, so the error message should be fine.